### PR TITLE
7 service model

### PR DESCRIPTION
**Pull request by hutsulyk**
Task "Service model"

### CODE REVIEW

File: moninag/service/admin.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```
Comment (lhalam) [inline moninag/service/admin.py]: delete this comment

File: moninag/service/tests.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```
Comment (lhalam) [inline moninag/service/tests.py]: delete this comment

File: moninag/service/views.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
+
+# Create your views here.
```
Comment (lhalam) [inline moninag/service/views.py]: delete this comment

**Review by byshovets — CHANGES_REQUESTED** [review]
I would like to see readable commit-history like:
* Created Service model
* Add deployment steps for Ubuntu
* Add Sublime and PyCharm artifacts to .gitignore

**Review by byshovets — APPROVED** [review]

**Pull request merged**